packages/workflow/src/NodeErrors.ts

```diff
@@ -480,6 +480,9 @@ export class NodeApiError extends NodeError {
 		if (STATUS_CODE_MESSAGES[this.httpCode]) {
 			this.description = this.updateDescription(this.message, this.description);
 			this.message = STATUS_CODE_MESSAGES[this.httpCode];
+
+			if (this.httpCode === '404') this.severity = 'warning';
```

Comment:
we have a very similar check on like 361 for 5xx. Maybe we should update it to include 4xx (not just 404).

Reply:
```ts
else if (httpCode?.charAt(0) !== '5') this.severity = 'warning'
```

So in fact 404 and 4XX should already be covered but it's still getting reported...